fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceGroup.java

```diff
@@ -181,12 +223,13 @@ public void addResources(List<IDatabaseObject> group, String prefix) {
                 .addTimestampColumn(    LAST_UPDATED,              false)
                 .addCharColumn(           IS_DELETED,           1, false)
                 .addBlobColumn(                 DATA,  2147483647,  10240,   true)
-                // Start Backwards Compatibility 
+                // Start Backwards Compatibility
                 .addVarcharColumn(TX_CORRELATION_ID, 36, true)
                 .addVarcharColumn(CHANGED_BY, 64, true)
-                .addVarcharColumn(CORRELATION_TOKEN, 36, true) 
+                .addVarcharColumn(CORRELATION_TOKEN, 36, true)
                 .addVarcharColumn(REASON, 255, true)
                 .addVarcharColumn(SERVICE_ID, 32, true)
+                .addVarcharColumn(TENANT_ID, 36, true)
                 // End Backwards Compatibility
```

Comment:
out of curiosity why is TENANT_ID added back?  are we setting this on INSERT?

Reply:
yes, it's in the storedprocedure, I choose to add it instead of removing it because existing envs.

Reply:
I suspect changing the stored procedure is probably preferred. Is the TENANT_ID actually used?

Reply:
got it - because it's not null

Reply:
Looks good!

Reply:
I don't really think it's used, I think we should further clean the codes later, I kept it there just because the storeprocedure with tenant_id works well with our cloud deployment, and we don't have a clear path or db schema migration yet, so just want to play safe for now.